docs/source/how_to/how_to_criterion_function.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>that would also work <strong>with</strong> scipy.optimize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/optimagic/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>